source/nodejs/tests/ui-tests/src/ui-tests.test.ts

```diff
@@ -50,4 +51,85 @@ describe("Mock function", function() {
 
         Assert.strictEqual(firstInputId, activeElementId);
     }));
+
+    test("Test actions are rendered and active below carousel", (async() => {
+        await testUtils.goToTestCase("v1.6/Carousel.HostConfig");
+
+        await testUtils.clickOnActionWithTitle("See more");
+
+        // wait half a second to wait for page refresh
+        await testUtils.delay(500);
```

Comment:
nit: can you make it a globally configurable property, so we can change the value in the future in one place? one test case for half a second is fine, but I'm worried in the future where we have hundreds of such tests.